packages/wrangler/src/api/pages/publish.tsx

```diff
@@ -95,9 +96,10 @@ export async function publish({
 		_redirects: string | undefined,
 		_routesGenerated: string | undefined,
 		_routesCustom: string | undefined,
+		_workerJSDirectory = false,
```

Comment:
Maybe call this something like `_workerJSIsDirectory` to make it clear that it's a boolean flag?